### PR TITLE
COB Timestamps

### PR DESCRIPTION
**Pull request by cloudhead**
*See commits*

### CODE REVIEW

File: radicle-cob/src/history.rs [hunk 1]
```diff
@@ -85,6 +87,16 @@ impl History {
             .unwrap_or_default()
     }
 
+    /// Get the current history timestamp.
+    /// This is the latest timestamp of any tip.
+    pub fn timestamp(&self) -> Clock {
```
Comment (FintanH) [inline radicle-cob/src/history.rs:92]: The dangers of `type` :smile: 
```suggestion
    pub fn timestamp(&self) -> Timestamp {
```

File: radicle-crdt/src/clock.rs [hunk 2]
```diff
@@ -54,3 +57,59 @@ impl Bounded for Lamport {
...
+    seconds: u64,
+}
+
+impl Physical {
+    pub fn new(seconds: u64) -> Self {
+        Self { seconds }
+    }
+
+    pub fn now() -> Self {
+        #[allow(clippy::unwrap_used)] // Safe because Unix was already invented!
```
Comment (FintanH) [inline radicle-crdt/src/clock.rs:74]: :smile:

**Review by FintanH — APPROVED** [review]

File: radicle-cob/src/history.rs (hunk 1 above)
Comment (cloudhead) [reply radicle-cob/src/history.rs:92]: Hahaha indeed.

File: radicle-crdt/src/clock.rs (hunk 2 above)
Comment (cloudhead) [reply radicle-crdt/src/clock.rs:74]: Hehe.

**Pull request merged**